modules/ROOT/pages/policies-landing-page.adoc

```diff
@@ -4,10 +4,30 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: policy, custom, ootb, offline
 
-A policy extends the functionality of an API and enforces certain capabilities such as security. A policy can control access and traffic without modification to the API implementation. Custom policies can provide other capabilities, such as encrypting and logging.
+Policies enable you to enforce regulations to help manage security, control traffic, and improve adaptability of your APIs. For example, a policy can control authentication, access, allotted consumption, and service level access (SLA). 
+
+You can implement all these regulations with no modification to the code implementation. Mulesoft provides ready-to-use default policies that are shipped with the product. Additionally, you can create custom policies based on your specific business requirements.
+
+You can apply policies to any HTTP-based APIs, such as:
+
+* An APIkit project
++
+For example, deploy the APIkit project to Anypoint Platform using API Autodiscovery and apply a policy.
++
+* An API running on CloudHub
++
+Design an API on Anypoint Platform, configure a proxy for Cloudhub, and apply a policy.
+* An API deployed to a private or cloud-based Mule Runtime 3.8.x or later
```

Comment:
```suggestion
* An API deployed to a private or cloud-based Mule runtime engine 3.8.x or later
```
GLOBALLY replace all other wordings with "Mule runtime engine" or "Mule" (after introducing "Mule runtime engine (Mule)", in all writing.

Reply:
Done! Thanks!